lib/seek/samples/attribute_type_handlers/cv_attribute_type_handler.rb

```diff
@@ -5,7 +5,7 @@ class CVAttributeTypeHandler < BaseAttributeHandler
         class MissingControlledVocabularyException < AttributeHandlerException; end
 
         def test_value(value)
-          fail "'#{value}' is not included in the controlled vocabulary" unless controlled_vocab.includes_term?(value)
+          fail "'#{value}' is not included in the controlled vocabulary" unless controlled_vocab.includes_term?(value) || controlled_vocab.source_ontology.present?
```

Comment:
this looks wrong. Even if the CV is from an ontology, the value should still be present in the list of terms to be valid

Reply:
What do you suggest if we want to accept inputs that are not in the list of **terms**?
**Requirement**: The users should be able to either choose from terms or input a **custom** value.

Reply:
if it is not in the list of terms, then it cannot be resolved to it's associated URI, which will then lead to other problems. It also prevents it behaving as a controlled vocabulary.

If it is an essential requirement then it would be better to have a flag on the SampleControlledVocab, which can be used to override the normal behaviour where the user wants to allow arbitary terms.